learning_wavelets/evaluate/exact_recon_unet_evaluate.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+import numpy as np
+import tensorflow as tf
+from tqdm import tqdm
+
+from runstats import Statistics
+from skimage.measure import compare_psnr, compare_ssim
```

Comment:
For this part, I advise you to read the [pep8 guidelines on the import order](https://www.python.org/dev/peps/pep-0008/#imports)

Reply:
also I think that some of these imports are not needed anymore